data/dataexchange/src/main/java/tech/pegasys/teku/data/eraFileFormat/EraFile.java

```diff
@@ -111,10 +240,25 @@ final void close() throws IOException {
   }
 
   public static void main(String[] args) throws IOException {
-    final Path p = Path.of(args[0]);
-    final EraFile f = new EraFile(p);
-    f.readEraFile();
-    f.printStats();
-    f.close();
+    Spec spec = null;
+    SignedBeaconBlock lastBlock = null;
+    for (String filename : args) {
+      final Path p = Path.of(filename);
+      if (spec == null) {
+        final String network =
+            Arrays.stream(p.getFileName().toString().toLowerCase(Locale.ROOT).split("-", 2))
+                .findFirst()
+                .orElseThrow();
+        System.out.println("Loading network: " + network);
+        spec = SpecFactory.create(network);
```

Comment:
we could have this in a private method like `getNetworkFromEraFilename` and we can call it once outside the loop using the first file from the arg

Reply:
Fair, I guess I was more thinking this is just a test file but I'll do that and add a usage if there's no args.

Reply:
Ah yes...:)